server/http_handler_test.go

```diff
@@ -688,7 +688,7 @@ func (ts *HTTPHandlerTestSuite) TestGetSchema(c *C) {
 	var dbs []*model.DBInfo
 	err = decoder.Decode(&dbs)
 	c.Assert(err, IsNil)
-	expects := []string{"information_schema", "mysql", "performance_schema", "test", "tidb"}
+	expects := []string{"information_schema", "metric_schema", "mysql", "performance_schema", "test", "tidb"}
```

Comment:
I am not sure whether `metric_schema` is the proper name, or is `metrics_schema` better? @bb7133